_2017_pages/cfp.html

```diff
@@ -15,102 +15,273 @@ <h2 class="first">Key Dates</h2>
-        of course, they are free to participate in the discussions with the audience outside of this Q&A session as well.
+      <p>
+      This conference will have two tracks:
+      <ul>
+        <li>a "Conjugate Model" track for folks just getting started out,</li>
+        <li>and a "No U Turn Sampler" track for folks that are already  warmed up with
+          Bayesian Methods.</li>
+      </ul>
 
-      </p>
+      First time speakers are welcomed and encouraged. Please feel free to reach
+      out with questions about submitting a proposal, you can find us at the <a href="https://discourse.pymc.io/t/pymcon-2020-call-for-proposals-is-open/5567">PyMC discourse</a>.  </p>
 
-      <h3 class="first">Tutorials</h3>
-      <p class=normal-text>
-        Tutorials are dedicated to introducing, promoting, and advancing specific techniques/practices in computational Bayesian statistics with PyMC.
-        Each tutorial, also to be recorded, is one hour long and should cover a topic more in depth than a traditional talk.
-        Tutorial facilitators are responsible for preparing interactive and carefully designed materials,
-        including hands-on exercises that help reinforce the concepts discussed in their topics.
-        We will provide the platform (most likely Slack) in which tutorial facilitators can communicate and share materials with their audience prior to the conference.
-      </p>
-
-      <h3 class="first">Let's Build a Model</h3>
-      <p class=normal-text>
-        We are excited to introduce a new conference talk format called Let’s Build a Model (LBAM).
-        While sharing many common characteristics with tutorials,
-        LBAM talks aim to showcase the exciting (and sometimes messy!) nature of Bayesian modeling.
-        In a LBAM talk, the presenter should first establish a specific modeling goal and then proceed to build that Bayesian model from scratch using PyMC (e.g., in a Jupyter notebook).
-        A LBAM talk is also a pre-recorded video and should be one hour long.
-        We know Bayesian modeling is a considerably involved process and can therefore be daunting to new users.
-        With LBAM talks, we aim to address this problem by encouraging presenters to be as thorough as possible,
-        while making sure to include relevant “tricks of the trade” that they find useful in their own experience.
-      </p>
-
-      <h2 class="first">What we look for in a submission</h2>
-      <p class = "normal-text">
-        More to come
+      <p>
+      There are three presentation formats:
+      <dl>
+        <dt><a class="reference internal" href="#talks">Talks</a></dt>
+        <dd>30-40 min video about PyMC tricks, best practices, related library, etc.</dd>
+        <dt><a class="reference internal" href="#tutorials">Tutorials</a></dt>
+        <dd>60 min video + engaging material diving into a spedific PyMC related topic.</dd>
```

Comment:
Small typo, I can fix. 

What do you think about adding ranges for the building example model as well. 30 mins to as long as you want?

Reply:
I would add an upper bound, even if it's 2 hours, even being async